README.md

```diff
@@ -2,105 +2,17 @@
 
 ## About
 
-This course is designed to be the absolute best starting point for Web Developers looking to learn Web3 Development. Solana is the ideal network for starting your Web3 journey because of its high speed, low cost, energy efficiency, and more.
+This course is designed to be the absolute best starting point for web developers looking to learn web3 development. Solana is the ideal network for web3 because of its high speed, low transaction costs, energy efficiency, and large developer community.
 
-This repository represents an ongoing project. We'll update the list of available lessons in the [Get Started](#get-started) section below as they're completed. If you'd like to help out, have a look [how you can contribute](#contribute-to-the-course).
+The course is currently hosted at https://soldev.app. 
```

Comment:
My only suggestion would be to change the URL to https://soldev.app/course both here and on the next line. Otherwise people will get lost in all the other content on Soldev.

Reply:
Thanks and done!